cmd/monaco/integrationtest/v2/references_test.go

```diff
@@ -0,0 +1,245 @@
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/testutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/api"
+	config "github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2"
+	refParam "github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/parameter/reference"
+	valueParam "github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2/parameter/value"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/manifest"
+	project "github.com/dynatrace/dynatrace-configuration-as-code/pkg/project/v2"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"regexp"
+	"testing"
+)
+
+func TestReferences(t *testing.T) {
+
+	envs := []string{"classic_env", "platform_env"}
+
+	tests := []struct {
+		project      string
+		downloadOpts []string
+		validate     func(t *testing.T, ctx TestContext, confsPerType project.ConfigsPerType)
+	}{
+		{
+			project: "classic-apis",
+			downloadOpts: []string{
+				"-a", "alerting-profile,notification,management-zone",
+			},
+			validate: func(t *testing.T, ctx TestContext, confsPerType project.ConfigsPerType) {
+				managementZone := findConfig(t, confsPerType, "management-zone", "zone_"+ctx.suffix)
+				profile := findConfig(t, confsPerType, "alerting-profile", "profile_"+ctx.suffix)
+				notification := findConfig(t, confsPerType, "notification", "notification_"+ctx.suffix)
+
+				assertRefParamFromTo(t, profile, managementZone)
+				assertRefParamFromTo(t, notification, profile)
+			},
+		},
+		{
+			project: "settings",
+			downloadOpts: []string{
+				"-s", "builtin:problem.notifications,builtin:management-zones,builtin:alerting.profile",
+			},
+			validate: func(t *testing.T, ctx TestContext, confsPerType project.ConfigsPerType) {
+				managementZone := findSetting(t, confsPerType, "builtin:management-zones", "zone_"+ctx.suffix, "name")
+				profile := findSetting(t, confsPerType, "builtin:alerting.profile", "profile_"+ctx.suffix, "name")
+				notification := findSetting(t, confsPerType, "builtin:problem.notifications", "notification_"+ctx.suffix, "displayName")
+
+				assertRefParamFromTo(t, profile, managementZone)
+				assertRefParamFromTo(t, notification, profile)
+			},
+		},
+		{
+			project: "settings-with-classic-mngt-zone",
+			downloadOpts: []string{
+				"-a", "management-zone",
+				"-s", "builtin:problem.notifications,builtin:alerting.profile",
+			},
+			validate: func(t *testing.T, ctx TestContext, confsPerType project.ConfigsPerType) {
+				managementZone := findConfig(t, confsPerType, "management-zone", "zone_"+ctx.suffix)
+				profile := findSetting(t, confsPerType, "builtin:alerting.profile", "profile_"+ctx.suffix, "name")
+				notification := findSetting(t, confsPerType, "builtin:problem.notifications", "notification_"+ctx.suffix, "displayName")
+
+				assertRefParamFromTo(t, profile, managementZone)
+				assertRefParamFromTo(t, notification, profile)
+			},
+		},
+		{
+			project: "config-with-mngt-zone",
```

Comment:
nit: `classic-with-settings-mngt-zone` would better fit the naming scheme you've applied to the other cases and make it clearer what this case does